### DISCUSSION TITLE
如何在定时任务中，由机器人提问并根据用户的回答发送不同的信息

### DISCUSSION THREAD

**Discussion by tiamat-love** [nonebot/discussions#222] [问答]
如何在定时任务中，由机器人提问并根据用户的回答发送不同的信息
想在require('nonebot_plugin_apscheduler').scheduler中实现类似should_paused的功能

**Comment by yanyongyu** [comment]
定时任务里通过bot发送消息，然后创建一个临时的matcher `on_message(priority=0, temp=True, permission=USER(QQ号), handlers=[处理函数])`
